packages/block-library/src/navigation-menu-item/editor.scss

```diff
@@ -64,29 +64,14 @@
 }
 
 .wp-block-navigation-menu-item {
-	&:not(.is-editing) {
-		font-family: inherit;
-		font-size: $text-editor-font-size;
-		font-weight: bold;
-	}
-
-	&.is-editing .components-text-control__input {
-		width: inherit;
-		background-color: inherit;
-		color: inherit;
-
-		&:focus {
-			color: inherit;
-		}
-	}
-
 	&.is-editing,
 	&.is-selected {
 		min-width: 20px;
 	}
 
 	.block-editor-rich-text__editable.is-selected:not(.keep-placeholder-on-focus):not(:focus) [data-rich-text-placeholder]::after {
 		display: inherit;
+		min-width: 30px;
```

Comment:
why do we need to increase the width here?
It seems to look ok, at least in my testing site.

![image](https://user-images.githubusercontent.com/77539/68472795-12fc4680-0200-11ea-80f2-cebf97a5de2e.png)

Reply:
it might be a mistake rebasing. let me check

Reply:
removed.

Reply:
🙇